pkg/asset/installconfig/gcp/client.go

```diff
@@ -343,3 +352,44 @@ func (c *Client) getServiceUsageService(ctx context.Context) (*serviceusage.Serv
 func (c *Client) GetCredentials() *googleoauth.Credentials {
 	return c.ssn.Credentials
 }
+
+func (c *Client) getPermissions(ctx context.Context, project string, permissions []string) ([]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	service, err := c.getCloudResourceService(ctx)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to get cloud resource manager service")
+	}
+
+	projectsService := cloudresourcemanager.NewProjectsService(service)
+	rb := &cloudresourcemanager.TestIamPermissionsRequest{Permissions: permissions}
+	response, err := projectsService.TestIamPermissions(project, rb).Context(ctx).Do()
```

Comment:
I wonder if `TestIamPermissions` requires a new permission in order to run?

Reply:
According to https://cloud.google.com/resource-manager/reference/rest/v1/projects/testIamPermissions, there are no permissions required to run this API call